content/teams/BlockCoder/index.md

```diff
@@ -5,7 +5,7 @@ name: "BlockCoder"
 github-link: "https://github.com/ajinkyabrahmankar"
 description: "Blockchain enthusiast"
 Team-Quote: ""
```

Comment:
mention team_Quote

Reply:
Yes , working on it

Reply:
The issue has been addressed and worked on